certbot_zimbra.sh

```diff
@@ -7,7 +7,7 @@ AGREE_TOS=""
 NO_NGINX="no"
 RENEW_ONLY="no"
 NEW_CERT="no"
-WEBROOT="/opt/zimbra/data/nginx/html"
+WEBROOT="/opt/zimbra/nginx/html"
```

Comment:
I don't have `/opt/zimbra/nginx/html` in 8.8.10. Why did you change this?

Reply:
Fixed , it is what I have in zimbra (8.6 I guess)

Reply:
Yes, it was probably at that path in 8.6, but it's irrelevant for one or the other so IIRC I've used just the new one for all versions